dex-it/src/test/scala/com/wavesplatform/it/sync/api/ws/WsPingPongTestSuite.scala

```diff
@@ -115,14 +115,14 @@ class WsPingPongTestSuite extends WsSuiteBase {
       wsac1.send(wsac2.pings.head) // send correct pong but from another connection
       wsac2.send(wsac1.pings.head) // send correct pong but from another connection
 
-      Thread.sleep(pongTimeout - 0.2.second)
+      Thread.sleep(pongTimeout - 0.3.second)
```

Comment:
why?

Reply:
Additional awaiting of WsInitial in `mkWsConnection` affects test